### ISSUE TITLE
Optional arguments are sent as undefined

### ISSUE THREAD

**Issue by Richard87** [rtk-incubator/rtk-query-codegen#16]
Hi!

I have this definition:
```
export type ApiCoursesJobsGetSubresourceApiArg = {
    id: string;
    /** The collection page number */
    page?: number;
    /** The number of items per page */
    itemsPerPage?: number;
};
```
When I run `const {data, ...response} = useApiCoursesJobsGetSubresourceQuery({id: 27, itemsPerPage: 100, page:1})` without `page` or `itemsPerPage`, they are included in the request as string values `"undefined"`, with a 400 Bad request error message from the API...

Does this bug belong in rtk-query-codegen or in rtk-query, or something I should fix on my end? :)

**Comment by msutkowski** [comment]
Looks like @kahirokunn [opened a PR for this earlier](https://github.com/rtk-incubator/rtk-query/pull/146). We'll just want to add test cases for that (sorry I missed them 😞) and then I'd recommend using the CSB build.

**Comment by msutkowski** [comment]
@Richard87 Can you confirm you're using `fetchBaseQuery` as the default generated baseQuery?

**Comment by msutkowski** [comment]
@Richard87 Would you mind trying this version of rtk-query in your project and reporting back? 

```sh
yarn add https://pkg.csb.dev/rtk-incubator/rtk-query/commit/8d6e1c9b/@rtk-incubator/rtk-query
```

**Comment by Richard87** [comment]
Hi! Yes, I'm using FetchBaseQuery

here is my generated file: https://gist.github.com/Richard87/c29f22ca2649afe237c73fcd9853195b (I replaced `-` and `:` with `_` in all names).

I can confirm at the commit over (https://pkg.csb.dev/rtk-incubator/rtk-query/commit/8d6e1c9b/@rtk-incubator/rtk-query) does not have the bug :)

**Comment by kahirokunn** [comment]
https://github.com/rtk-incubator/rtk-query-codegen/issues/16#issuecomment-758805186

**Comment by msutkowski** [comment]
Alright, released `0.2`, please upgrade :)
